LICENSE

```diff
@@ -19,78 +19,3 @@ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
 ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-
-Subcomponents
```

Comment:
For historical context, at the time this section of the license was written, the client code not only lived in the h repo, but also had vendored copies of many of the major libraries in the source tree, rather than getting them via a package manager.